rfcs/2023-11-08-font-loading-strategy.md

```diff
@@ -204,16 +214,16 @@ The performance of variable font vs static font depends on how many weights and
 These are the weights and styles being used right now
 
 **Tasa Orbiter:** 400, 500, 600
-**Inter:** 400, 600
+**Inter:** 400, 500, 600
 
 **Conclusion**
 
-We'll use TASA Orbiter as variable font and Inter as static font based on which is the lightest version.
+We'll be using variable fonts in both Tasa and Inter based on the size below-
 
-|                  | **Static**                        | **Variable** |
-| ---------------- | --------------------------------- | ------------ |
-| **Inter**        | 70kb \* 2 weights = ✅ **~140kb** | ~200kb       |
-| **TASA Orbiter** | 30kb \* 3 weights = ~90kb         | ✅ **~32kb** |
+|                  | **Static**                 | **Variable**                   |
+| ---------------- | -------------------------- | ------------------------------ |
+| **Inter**        | 70kb \* 3 weights = ~210kb | ✅ **~215kb (80kb base file)** |
```

Comment:
when we say 80kb base file what does that exactly mean? does it mean because of unicode optimisation we'll only load this much or does that mean that we'll load 80kb and then load additional things as well?

Reply:
It means in most scenarios we'll load 80kb but in rare scenarios if we end up using some unicode from other file, we might end up loading more.